osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-monitoring.xsd

```diff
@@ -540,4 +540,103 @@
     </xsd:sequence>
   </xsd:complexType>
 
+  <!-- ActualPowerQuality -->
+
+  <xsd:element name="ActualPowerQualityRequest">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="tns:ActualPowerQualityRequestData">
+          <xsd:sequence>
+            <xsd:element name="DeviceIdentification" type="common:DeviceIdentification"/>
+          </xsd:sequence>
+        </xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:complexType name="ActualPowerQualityRequestData">
+    <xsd:complexContent>
+      <xsd:extension base="common:Query">
+        <xsd:sequence>
+          <xsd:element name="ProfileType" type="xsd:string"/>
+        </xsd:sequence>
+      </xsd:extension>
+    </xsd:complexContent>
+  </xsd:complexType>
+
+  <xsd:element name="ActualPowerQualityResponse">
+    <xsd:complexType>
+      <xsd:complexContent>
+        <xsd:extension base="tns:ActualPowerQualityResponseData"></xsd:extension>
+      </xsd:complexContent>
+    </xsd:complexType>
+  </xsd:element>
+
+  <xsd:complexType name="ActualPowerQualityResponseData">
+    <xsd:complexContent>
+      <xsd:extension base="common:Response">
+        <xsd:sequence>
+          <xsd:element name="ActualPowerQualityData" type="tns:ActualPowerQualityData"
+                       minOccurs="1"
+                       maxOccurs="1"/>
+        </xsd:sequence>
+      </xsd:extension>
+    </xsd:complexContent>
+  </xsd:complexType>
+
+  <xsd:complexType name="ActualPowerQualityData">
+    <xsd:sequence>
+      <xsd:sequence>
+        <xsd:element name="PowerQualityObjects" type="tns:PowerQualityObjects" minOccurs="1" maxOccurs="1"/>
+        <xsd:element name="ActualValues" type="tns:ActualValues" minOccurs="1" maxOccurs="1"/>
```

Comment:
It's not that I strongly oppose, but `minOccurs` and `maxOccurs` are set to their default values here explicitly. These assignments could be removed.

Reply:
removed